### PR TITLE
Refactoring and docs

### PR DESCRIPTION
**Pull request by zrzka**
* Travis CI
  * Build PRs & `master` branch only
* Copy the book content to the `lib.rs`
* Properly document public API (docs.rs)
* Update README & CHANGELOG

### CODE REVIEW

**Comment by zrzka** [comment]
@TimonPost no longer WIP, if it will pass Travis CI, it's ready.

File: README.md [hunk 1]
```diff
@@ -3,100 +3,72 @@
...
 ```
 
-### Useful Links
+## Future
 
-- [Documentation](https://docs.rs/crossterm_screen/)
-- [Crates.io](https://crates.io/crates/crossterm_screen)
-- [Book](https://crossterm-rs.github.io/crossterm/docs/screen.html)
-- [Examples](https://github.com/crossterm-rs/examples)
+> The `crossterm_screen` crate code will be moved to the `crossterm` crate (it's reexported there now).
```
Comment (TimonPost) [inline README.md]:
```suggestion
> The `crossterm_screen` crate code will be moved to the `crossterm` crate (it's re-exported there now).
```

File: src/lib.rs [hunk 2]
```diff
@@ -1,7 +1,69 @@
-#![deny(unused_imports)]
+#![deny(unused_imports, unused_must_use)]
+
+//! # Screen
+//!
+//! The `crossterm_screen` crate provides a functionality to work with the terminal screen.
```
Comment (TimonPost) [inline src/lib.rs]:
```suggestion
//! The `crossterm_screen` crate provides the functionality to work with the terminal screen.
```

**Review by TimonPost — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zrzka) [reply README.md]: Fixed.

File: src/lib.rs (hunk 2 above)
Comment (zrzka) [reply src/lib.rs]: Fixed.

**Comment by zrzka** [comment]
Waiting for CI, then will merge.

**Pull request merged**